addon/templates/components/polaris-label.hbs

```diff
@@ -0,0 +1,9 @@
+<div class="Polaris-Label">
```

Comment:
missing some [hidden class](https://github.com/Shopify/polaris/blob/master/src/components/Label/Label.tsx#L20) here

Reply:
@vladucu good catch, thanks! Fixed in #151 👍